lib/l10n-en_GB.js

```diff
@@ -192,7 +192,10 @@ exports.messages = {
     // sotd/draft-stability
     'sotd.draft-stability.not-found': 'No stability paragraph found.',
     // sotd/pp
-    'sotd.pp.no-pp': 'Required patent policy paragraph not found.',
+    'sotd.pp.no-pp':
+        'Required patent policy paragraph not found. Expected text related to patent policy requirements: <blockquote>${expected}</blockquote>',
+    'sotd.pp.no-pp-link':
+        'Required patent policy link not found in the paragraph: <blockquote>${expected}</blockquote>',
```

Comment:
To avoid confusion:
```suggestion
        'Required patent policy link not found. Expected text related to patent policy requirements: <blockquote>${expected}</blockquote>',
```